client/keystore/src/testing.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
```

Comment:
Please remove the file.

What you want is the keystore from `sp-core/testing/Keystore`.

Reply:
ah thanks